src/applications/claims-status/components/WIP.jsx

```diff
@@ -1,19 +1,18 @@
 import React from 'react';
 
 const WIP = () => (
-  <va-alert status="warning">
-    <h1 slot="headline" className="vads-u-font-size--h3">
-      We’re still working on this feature
-    </h1>
-    <p>
-      We’re rolling out the VA Claim Letters page in stages. It’s not quite
-      ready yet. Please check back again soon.
-    </p>
-    <p>
-      <a href="/track-claims/your-claims">
-        Return to VA Claims and Appeals page
-      </a>
-    </p>
+  <va-alert close-btn-aria-label="Close notification" status="warning" visible>
+    <h2 slot="headline">
+      Decision letters aren’t available to download right now.
+    </h2>
+    <div>
+      <p className="vads-u-margin-y--0 vads-u-margin-bottom--0">
```

Comment:
If you don't need margins, you probably can just get rid of the `<p>` tag and the `<div>` tag. If you do need the div/p, then the `vads-u-margin-bottom--0` is redundant as the `vads-u-margin-y--0` should cover both top and bottom margins